examples/fedpca_examples/perform_pca/config.yaml

```diff
@@ -0,0 +1,12 @@
+# Parameters that describe server
+n_server_rounds: 1 # The number of rounds to run FL
+
+# Parameters that describe clients
+n_clients: 4 # The number of clients in the FL experiment
+batch_size: 32 # The batch size for client training
+
+low_rank: False
```

Comment:
Some brief inline comments (along the same lines as the readme, but can be less informative) would be useful.